java-diff-utils/src/main/java/com/github/difflib/patch/AbstractDelta.java

```diff
@@ -70,6 +70,18 @@ protected VerifyChunk verifyAntApplyTo(List<T> target) throws PatchFailedExcepti
     
     protected abstract void restore(List<T> target);
     
+    /**
+     * Apply patch fuzzy.
+     *
+     * @param target the list this patch will be applied to
+     * @param fuzz the count of elements which ignores prefix and suffix
```

Comment:
I have a hard time understanding this parameter description. Please try rephrasing it and/or embed an example in the Javadoc.